setup.py

```diff
@@ -1,58 +0,0 @@
-#!/usr/bin/env python
-
-from setuptools import setup, find_packages
-from distutils.core import Extension
-
```

Comment:
Again, newbie question. I understand that poetry uses pyproject.toml to replace setup.py, requirements.txt, etc. However how do I update the pip package once a new version is released? I use `twine upload dist/*` to do that currently.